### PR TITLE
Switch webserver

### PR DESCRIPTION
**Pull request by ArturMinnullin**
Switch webserver to Puma, use dynamic database name. Also it solves https://github.com/fs/rails-base-api/issues/163.

### CODE REVIEW

**Comment by timurvafin** [comment]
:ok:

File: Gemfile.lock [hunk 1]
```diff
@@ -81,17 +81,20 @@ GEM
...
+    columnize (0.3.6)
     concurrent-ruby (1.0.0)
     crack (0.4.2)
       safe_yaml (~> 1.0.0)
-    daemons (1.1.9)
     database_cleaner (1.2.0)
+    debugger-linecache (1.2.0)
     decent_exposure (2.3.1)
-    devise (3.5.6)
+    devise (3.5.1)
```
Comment (arkadiybutermanov) [inline Gemfile.lock]: The version is downgraded
See https://semaphoreci.com/fs/rails-base-api/branches/pull-request-187/builds/2

**Comment by ArturMinnullin** [comment]
@arkadiybutermanov ready to merge

**Comment by timurvafin** [comment]
@ArturMinnullin could you please sync your branch with master?

**Comment by timurvafin** [comment]
@ArturMinnullin could you please sync your branch with `master`?

**Comment by vladimirbazhanov** [comment]
dublicate of #225

**Pull request closed without merging**